manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.installer.yaml

```diff
@@ -0,0 +1,49 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CloudFoundry.CLI.v8
+PackageVersion: 8.7.2
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli_8.7.2_winx64.zip
+  InstallerSha256: F4FCD6D166C82ACBB7796B90746BFB731A5C8A1998C13F373002A3AD08062471
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli_8.7.2_win32.zip
+  InstallerSha256: 3125EF78DEF460D4A9C8C5DEACAF4F291F10BF484E9C96D493F0FFA7DE30B5BF
+- Architecture: x64
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_winx64.zip
+  InstallerSha256: 489BBBF3DB174195C430E23F4A96BC27951AE8FE54159BF5042B431180F332AD
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.2
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
+- Architecture: x86
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_win32.zip
+  InstallerSha256: 6CECF7ADC3C8BC7A324B536B709D70A5DF51309F14EE076DD64C6A7D512F94AE
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.2
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
```

Comment:
```suggestion
```

Same feedback as https://github.com/microsoft/winget-pkgs/pull/136754#discussion_r1468808513. DisplayVersion is not `v8.7.2` here